test/integration/windows/windows_test.go

```diff
@@ -76,6 +83,74 @@ var _ = Describe("Windows Integration Test", func() {
 		frameWork.NSManager.DeleteAndWaitTillNamespaceDeleted(ctx, namespace)
 	})
 
+	Describe("configMap enable-windows-ipam tests", func() {
+		var testPod *v1.Pod
+		var createdPod *v1.Pod
+		BeforeEach(func() {
+			testPod, err = manifest.NewWindowsPodBuilder().Container(testerContainer).Build()
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		Context("when configmap created", func() {
+			JustBeforeEach(func() {
+				// Update configmap for tests
+				configMapWrapper.UpdateConfigMap(frameWork.K8sClient, ctx, configMap, data)
+
+			})
+			JustAfterEach(func() {
+				// restore the configmap after each test
+				data = map[string]string{config.EnableWindowsIPAMKey: "true"}
+				configMapWrapper.UpdateConfigMap(frameWork.K8sClient, ctx, configMap, data)
+			})
+
+			Context("when enable-windows-ipam is True", func() {
+				It("pod should be running and have resourceLimit injected", func() {
+					createdPod, err = frameWork.PodManager.CreateAndWaitTillPodIsCompleted(ctx, testPod)
```

Comment:
I hope the container is not a server, if it is then this condition will never return. Maybe you wanted to use CreateAndWaitTillPodIsRunning here?